.github/workflows/check.yml

```diff
@@ -34,3 +35,13 @@ jobs:
     #     <remote host>: <ssh-ed25519>
     # - run: nix run --show-trace . -- ssh-show-ed25519 <remote host>
 
+    - name: Bash format check
+      run: |
+        # waiting for follows fix patch
+        # nix flake lock --update-input beautysh
+        nix -Lv develop -c format-bash
+
+    - name: Auto Commit format changes
+      uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        commit_message: "Update format bash"
```

Comment:
I do not think we should make this an auto-commit action, for the noise it might signify, rather add it for a pre commit hook

Reply:
Agree, let me reset it.